t/t-lock.sh

```diff
@@ -249,7 +249,33 @@ begin_test "lock with .gitignore"
   echo "*.txt" > .gitignore
   git add .gitignore
   git commit -m ".gitignore: ignore 'a.txt'"
+  rm -f a.txt && git checkout a.txt
+  assert_file_writeable a.txt
+)
+end_test
+
+
+begin_test "lock with .gitignore and lfs.lockignoredfiles"
+(
+  set -e
+
+  reponame="lock-with-gitignore-and-ignoredfiles"
+  setup_remote_repo_with_file "$reponame" "a.txt"
+  clone_repo "$reponame" "$reponame"
+
+  git config lfs.lockignorefiles true
+  echo "*.txt filter=lfs diff=lfs merge=lfs -text lockable" > .gitattributes
+
+  git add .gitattributes
+  git commit -m ".gitattributes: mark 'a.txt' as lockable"
+
   rm -f a.txt && git checkout a.txt
   refute_file_writeable a.txt
+
+  echo "*.txt" > .gitignore
+  git add .gitignore
+  git commit -m ".gitignore: ignore 'a.txt'"
+  rm -f a.txt && git checkout a.txt
+  assert_file_writeable a.txt
```

Comment:
If I'm reading this option correctly, shouldn't `a.txt` be locked due to the `lfs.lockignorefiles=true`?

Reply:
Thanks for catching this. I typoed the config option a couple lines earlier.